clang/include/clang/Sema/DeclSpec.h

```diff
@@ -353,7 +356,7 @@ class DeclSpec {
   unsigned ConstrainedAuto : 1;
 
   // type-qualifiers
-  unsigned TypeQualifiers : 5;  // Bitwise OR of TQ.
+  unsigned TypeQualifiers : 8;  // Bitwise OR of TQ.
```

Comment:
Explanation:
TypeQualifiers are defined under DeclSpec (First phase of parsing qualifiers) and QualType (2nd phase)
I have expanded TQ under DeclSpec but NOT under QualType as doing the second was rabbit hole and required major changes, due to its interweaving with 3-Bit FastMask (expanding this in size is extremely complex)